region/norfair/west.json

```diff
@@ -897,16 +1006,18 @@
                     {"and": [
                       "Super",
                       {"cost": [
-                        {"heatFrames": 150}
+                        {"heatFrames": 150},
+                        {"ammo": [
+                          {
+                            "type": "Super",
```

Comment:
I figure that should be the ammo to kill a Fune

Reply:
How would you want it represented?

Reply:
Same as all other ammo counts for killing enemies, which I guess we haven't really confirmed yet.